tsconfig.json

```diff
@@ -0,0 +1,90 @@
+      "@loaders.gl/loader-utils/*": ["modules/loader-utils/src/*"],
+      "@loaders.gl/loader-utils/test": ["modules/loader-utils/test"],
+      "@loaders.gl/polyfills": ["modules/polyfills/src"],
+      "@loaders.gl/polyfills/*": ["modules/polyfills/src/*"],
+      "@loaders.gl/polyfills/test": ["modules/polyfills/test"],
+
+      "@loaders.gl/3d-tiles/*": ["modules/3d-tiles/src/*"],
+      "@loaders.gl/3d-tiles/test": ["modules/3d-tiles/test"],
+      "@loaders.gl/arrow/*": ["modules/arrow/src/*"],
+      "@loaders.gl/arrow/test": ["modules/arrow/test"],
+      "@loaders.gl/basis/*": ["modules/basis/src/*"],
+      "@loaders.gl/basis/test": ["modules/basis/test"],
+      "@loaders.gl/csv/*": ["modules/csv/src/*"],
+      "@loaders.gl/csv/test": ["modules/csv/test"],
+      "@loaders.gl/draco/*": ["modules/draco/src/*"],
+      "@loaders.gl/draco/test": ["modules/draco/test"],
+      "@loaders.gl/images/*": ["modules/images/src/*"],
+      "@loaders.gl/images/test": ["modules/images/test"],
+      "@loaders.gl/gltf/*": ["modules/gltf/src/*"],
+      "@loaders.gl/gltf/test": ["modules/gltf/test"],
+      "@loaders.gl/json": ["modules/json/src"],
+      "@loaders.gl/json/*": ["modules/json/src/*"],
+      "@loaders.gl/json/test": ["modules/json/test"],
+      "@loaders.gl/kml/*": ["modules/kml/src/*"],
+      "@loaders.gl/kml/test": ["modules/kml/test"],
+      "@loaders.gl/i3s/*": ["modules/i3s/src/*"],
+      "@loaders.gl/i3s/test": ["modules/i3s/test"],
+      "@loaders.gl/las/*": ["modules/las/src/*"],
+      "@loaders.gl/las/test": ["modules/las/test"],
+      "@loaders.gl/mvt/*": ["modules/mvt/src/*"],
+      "@loaders.gl/mvt/test": ["modules/mvt/test"],
+      "@loaders.gl/obj/*": ["modules/obj/src/*"],
+      "@loaders.gl/obj/test": ["modules/obj/test"],
+      "@loaders.gl/pcd/*": ["modules/pcd/src/*"],
+      "@loaders.gl/pcd/test": ["modules/pcd/test"],
+      "@loaders.gl/ply/*": ["modules/ply/src/*"],
+      "@loaders.gl/ply/test": ["modules/ply/test"],
+      "@loaders.gl/potree/*": ["modules/potree/src/*"],
+      "@loaders.gl/potree/test": ["modules/potree/test"],
+      "@loaders.gl/tables/*": ["modules/tables/src/*"],
+      "@loaders.gl/tables/test": ["modules/tables/test"],
+      "@loaders.gl/tiles/*": ["modules/tiles/src/*"],
+      "@loaders.gl/tiles/test": ["modules/tiles/test"],
+      "@loaders.gl/terrain/*": ["modules/terrain/src/*"],
+      "@loaders.gl/terrain/test": ["modules/terrain/test"],
+      "@loaders.gl/wkt/*": ["modules/wkt/src/*"],
+      "@loaders.gl/wkt/test": ["modules/wkt/test"],
+      "@loaders.gl/zip/*": ["modules/zip/src/*"],
+      "@loaders.gl/zip/test": ["modules/zip/test"],
+
+      "test": ["test"]
+    }
+  },
+  "include":[
+    "modules/**/src/**/*",
+    "modules/**/test/**/*"
+  ],
+  "exclude":[
+    "modules/**/src/libs/*",
+
+    "modules/math/src/**/*",
+    "modules/math/test/**/*",
+
+    "modules/gltf/src/**/*",
+    "modules/gltf/test/**/*",
+
+    "modules/tiles/src/**/*",
```

Comment:
@xintongxia Should you be interested in helping out with typescript introduction, you can delete these exceptions one-by-one and fix the resulting linter errors.